server/src/app.js

```diff
@@ -1,9 +1,15 @@
 import express from 'express';
 
+import routes from './routes/index';
```

Comment:
the `/index` isn't required because the engine will pull it in automatically when you use `./routes`